azure/functions/_abc.py

```diff
@@ -300,3 +300,29 @@ def enqueued_time(self) -> typing.Optional[datetime.datetime]:
     @abc.abstractmethod
     def offset(self) -> typing.Optional[str]:
         pass
+
+class KafkaEvent(abc.ABC):
+
+    @abc.abstractmethod
+    def get_body(self) -> bytes:
+        pass
+    
+    @abc.abstractmethod
+    def key(self) -> typing.Optional[str]:
```

Comment:
This should be decorated with `@property` as well. Same applies for the properties below this as well since they do not follow method conventions.